### DISCUSSION TITLE
pulsar-daemon start standalone

### DISCUSSION THREAD

**Discussion by jinxiaoyi** [apache/pulsar#17550] [Q&A] [answered]
when i start pulsar2.10.1 use  ./pulsar-daemon start standalone    
3181 port is already use.   how can i  fix it.  may i change the port?

**Comment by michaeljmarshall** [comment, marked as answer]
This is the bookkeeper port. You can update it in the `conf/standalone.conf` by setting the following to a different port:

```conf
bookiePort=3181
```

**Reply by jinxiaoyi** [reply]
i try to update  conf/standalone.conf and  set bookiePort=3183 , but it work on 3181.

**Reply by jinxiaoyi** [reply]
i also update conf/bookkeeper.conf and set bookiePort=3183 , but it work on 3181  use  bin/pulsar-daemon start standalone
